src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,109 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private int size = 0;
+    private Node<K, V>[] hashTable = new Node[INITIAL_CAPACITY];
+    private int threshold = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
+    private int currentCapacity = INITIAL_CAPACITY;
```

Comment:
currentCapacity always == hashTable.length, we don't need this variable

Reply:
done